cogs/youtube.py

```diff
@@ -37,7 +37,8 @@ def search_youtube(user_input, number):
     DEVELOPER_KEY = token_youtube
 
     youtube = googleapiclient.discovery.build(
-        api_service_name, api_version, developerKey=DEVELOPER_KEY)
+        api_service_name, api_version, developerKey=DEVELOPER_KEY,
+        cache_discovery=False)
```

Comment:
google api was screaming WARNING logs to tell "use OAUTH blah blah blah or disable cache_discovery"